apps/emqx/src/emqx_types.erl

```diff
@@ -180,9 +181,10 @@
     anonymous => boolean(),
     cn => binary(),
     dn => binary(),
+    %% Following fields are never used in RPC calls.
```

Comment:
Nit: "The following fields"

Reply:
commented out for bpapi spec backward compatibility.

I will add the note